src/components/Collection/CleanDescription.tsx

```diff
@@ -2,21 +2,23 @@ import React, { useMemo } from 'react';
 import cx from 'classnames';
 
 // The question marks are there because people can't spell…
-const RemoveSummaryRegex = /\b(Sour?ce|Note|Summ?ary):([^\r\n]+|$)/mg;
+const CleanInfoLinesRegex = /\b((Modified )?Sour?ce|Note( [1-9])?|Summ?ary):(?!$)([^\r\n]+|$)/img;
 
-const RemoveBasedOnWrittenByRegex = /^(\*|\u2014) (based on|written by) ([^\r\n]+|$)/img;
-
-const RemoveBBCodeRegex = /\[i\](.*?)\[\/i\]/sg;
+// eslint-disable-next-line operator-linebreak -- Because dprint and eslint can't agree otherwise. Feel free to fix it.
+const CleanMiscLinesRegex =
+  /^(\*|\u2014 (adapted|source:?|summary|translated|written)|- (translated)|~ (adapted|description|summary|translated)) ([^\r\n]+|$)/img;
 
-const MultiSpacesRegex = /\s{2,}/g;
+// This accounts for an AniDB API bug since BBCode is not supposed to be there
+const CleanBBCodeRegex = /\[i\](?!"The Sasami|"Stellar|In the distant| occurred in)(.*?)\[\/i\]/isg;
+const CleanExtraBBCodeRegex = /(\[i\]|\[\/i\])/ig;
 
-const CleanMiscLinesRegex = /^(--|~) /sg;
+const CleanMultiEmptyLinesRegex = /\n{2,}/g;
 
-const CleanMultiEmptyLinesRegex = /\n{2,}/sg;
+const CleanMultiSpacesRegex = /\s{2,}/g;
```

Comment:
Should probably have the s flag, to not collapse newlines unintentionally, even if this is ran after the regex to reduce multiple new lines into a single new line.

Reply:
hmm, didn't think the s flag did anything there (added it anyways)

Reply:
I just thought it only affected the `.` wildcard no idea if that is true here vs what I'm used to using though

Reply:
With single line or DOTALL flag enabled `.` will match newline characters but you are not using `.` in these expressions so the flag does not have an effect.

Reply:
then I'm misremembering.